corehq/apps/app_manager/views/form_designer.py

```diff
@@ -76,6 +76,19 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
     })
 
+    scheduler_data_nodes = []
+    if getattr(module, 'has_schedule', False):
+        scheduler_data_nodes = [
+            "unscheduled_visit",
+            "current_visit_number",
+            "next_visit",
+            "next_visit_date",
+            "next_due",
+        ]
+        scheduler_data_nodes.extend([u"next_{}".format(form_.schedule_form_id)
+            for form_ in module.get_forms() if form_.schedule_form_id])
```

Comment:
This will get all of the forms in the module, which will work. However, the scheduler only injects the next_ properties for forms with a schedule in the same phase. You can do something like:
```py
forms = [f for f in form.get_phase().get_forms()
                 if getattr(f, 'schedule') and f.schedule.enabled]
```